tests/ttnn/unit_tests/test_squeeze.py

```diff
@@ -0,0 +1,27 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+
+import torch
+
+import ttnn
+
+
+@pytest.mark.parametrize(
+    "input_shape, dim, output_shape",
+    [
+        # ((1,256,1), -1, (1,256)),
```

Comment:
tracked by a separate ticket?

Reply:
This one is a reshape issue. We can discuss this. Unsqueeze prepares the new shape for reshape, and here reshape  complains 
`Unable to reshape given tensor!` Tracked in https://github.com/tenstorrent/tt-metal/issues/12715
Need to get this staffed asap, will discuss with @eyonland . I can work on it as well.